tests_backend/run_backend_tests.ts

```diff
@@ -72,13 +59,35 @@ async function genTempUpstashEnvFile(upstashToken: string, upstashUrl: string):
   const tempFilePath = await Deno.makeTempFile()
   const exampleEnvFile = await Deno.readTextFile('supabase/.env.exemple')
   const tempEnvFile = `${exampleEnvFile}\nREDIS_CONNECTION_TYPE=upstash\nREDIS_TOKEN=${upstashToken}\nREDIS_URL=${upstashUrl}`
+
+  const minioUrl = Deno.env.get('MINIO_URL')
+  if (minioUrl)
+    tempEnvFile.replace('host.docker.internal', minioUrl)
+
   await Deno.writeTextFile(tempFilePath, tempEnvFile)
   return tempFilePath
 }
 
+async function getEnvFile(redis: 'none' | 'local') {
+  const envFilePath = `supabase/.env.exemple${redis === 'local' ? '.redis' : ''}`
+
+  const minioUrl = Deno.env.get('MINIO_URL')
+  if (minioUrl) {
+    p.log.info('Minio URL is not null, creating a new env file...')
+    const tempEnvFile = await Deno.makeTempFile()
+    const readEnvFile = await Deno.readTextFile(envFilePath)
+    const finalFile = readEnvFile.replace('host.docker.internal', minioUrl)
+    await Deno.writeTextFile(tempEnvFile, finalFile)
+    return tempEnvFile
+  }
```

Comment:
Please also consider defining the string `supabase/.env.exemple` used on lines 60 and 72 as a `const` so that it is easy to keep track of. 

Though it is a valid spelling in French, I think it has been suggested before to fix the tiny typo in the file name by changing ".env.ex**e**mple" to ".env.ex**a**mple".

Reply:
If I understand correctly you want to make the `supabase/.env.exemple` read occur only once?